pkg/assert/node.go

```diff
@@ -28,7 +28,7 @@ func NodeAssertVersionTypeUpgrade(c customflag.FlagConfig) NodeAssertFunc {
 
 // assertVersion returns the NodeAssertFunc for asserting version
 func assertVersion(c customflag.FlagConfig) NodeAssertFunc {
-	fmt.Printf("Asserting Version: %s\n", c.InstallMode.Version)
+	shared.LogLevel("\ninfo", "Asserting Version: %s\n", c.InstallMode.Version)
```

Comment:
typo `\ninfo`?

Reply:
nope, its to break line before the log

Reply:
should probably use Enums/Constants for represent the log level

Reply:
could be a possible refactor in future